### DISCUSSION TITLE
Age Restriction

### DISCUSSION THREAD

**Discussion by doobydooby72** [TeamNewPipe/NewPipe#10818] [Questions]
Hi, i know that the new age verification stuff seems to have stopped a lot of 3rd party apps from viewing them. But id just like to mention that the age restriction bypass script on greasyforks site still works. So i figure there are still methods for workarounds. Thanks

Note: I couldnt find another discussion when i searched but im also not super good at using github, so sorry if i missed a relevant discussion that exists.

**Comment by opusforlife2** [comment]
@doobydooby72 Could you link an age restricted video that doesn't work for you?

**Reply by doobydooby72** [reply]
https://m.youtube.com/watch?v=UOXaBV8mD-I

I use to use 1 of the official red dead 2 gameplay trailers as a test but yt seems to have changed that so heres a rando video.

**Reply by opusforlife2** [reply]
@doobydooby72 Could you open a new issue? That link errors out for me as well.

**Reply by AudricV** [reply]
@doobydooby72 No need to do so. This video cannot be embedded on any website, and only embeddable age-restricted videos (on some websites or all websites) can be watched without logging-in currently.

**Reply by opusforlife2** [reply]
Then we should show a different error message than usual. Something like "Youtube refuses to show certain kinds of age-restricted videos even if you have toggled them on in settings. This is one such video."

**Reply by doobydooby72** [reply]
But it does work with that script i mentioned in a browser. Hence why i was saying that there seems to be workarounds that still work. Also i think in the countries that have the new age verification stuff they have made all restricted videos non embeddable, pretty sure i heard that somewhere at least.
